src/sentry/monitors/logic/mark_failed.py

```diff
@@ -189,6 +189,10 @@ def mark_failed_threshold(failed_checkin: MonitorCheckIn, failure_issue_threshol
     if monitor_muted:
         return True
 
+    # Do not create event/occurrence if we don't have a fingerprint
+    if not fingerprint:
+        return True
+
```

Comment:
I think we still want to send the signal here?

Reply:
Yeah sorry, didn't have a chance to update this yet